javascript-modules-library/src/core/server/components/AbsoluteArea.tsx

```diff
@@ -42,33 +56,40 @@ export function AbsoluteArea({
    * @default false
    */
   readOnly?: boolean | "children";
-  /** Ancestor level for absolute area - 0 is Home page, 1 first sub-pages, ... */
-  level?: number;
   /**
-   * Content type to be used to create the area
+   * Content node type to be used to create the area (if the node does not exist yet)
    *
+   * @deprecated Use {@link #nodeType} instead
    * @default jnt:contentList
    */
   areaType?: string;
-  /** The parameters to pass to the absolute area */
+  /**
+   * Content node type to be used to create the area (if the node does not exist yet)
+   *
+   * @default jnt:contentList
+   */
+  nodeType?: string;
+  /**
+   * Map of custom parameters that can be passed to the backend engine for advanced logic.
+   *
+   * @deprecated Not recommended
+   */
   parameters?: Record<string, unknown>;
 }>): React.JSX.Element {
-  const { renderContext } = useServerContext();
+  const { renderContext, currentNode } = useServerContext();
   return (
     // @ts-expect-error <unwanteddiv> is not a valid HTML element
     <unwanteddiv
       dangerouslySetInnerHTML={{
         __html: server.render.renderAbsoluteArea(
           {
             name,
-            areaView,
-            allowedTypes,
+            parent: parent ? parent : currentNode,
```

Comment:
Are we sure about this default value? It does not make much sense as it's not something absolute by default

Reply:
Yep, I like that